src/mscorlib/shared/System/Guid.cs

```diff
@@ -33,24 +33,63 @@ public partial struct Guid : IFormattable, IComparable, IComparable<Guid>, IEqua
         private byte _j; // Do not rename (binary serialization)
         private byte _k; // Do not rename (binary serialization)
 
-
         ////////////////////////////////////////////////////////////////////////////////
         //  Constructors
         ////////////////////////////////////////////////////////////////////////////////
 
         // Creates a new guid from an array of bytes.
-        //
         public Guid(byte[] b)
+        : this(new ReadOnlySpan<byte>(b ?? throw new ArgumentNullException(nameof(b))))
```

Comment:
Nit: this is a slightly unusual formatting; often the colon is at the end of the previous line, and then either the `this` is after it on the same line or on the next line but indented.